account/identity.go

```diff
@@ -138,6 +142,28 @@ func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Ident
 	return &native, errs.WithStack(err)
 }
 
+// Exists returns true|false where an object exists with an identifier
+func (m *GormIdentityRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "exists"}, time.Now())
+
+	queryStmt, err := m.db.CommonDB().Prepare(fmt.Sprintf(`
```

Comment:
@hectorj2f I always thought a prepared statement is only efficient when the statement is created is constructed once and then used multiple times. Here you're not reusing that statement, so I don't see a point.